vmngclient/session.py

```diff
@@ -344,6 +344,20 @@ def get_virtual_session_id(self, tenant_id: str) -> str:
         response = self.post(url_path)
         return response.json()["VSessionId"]
 
+    def logout(self) -> vManageResponse:
+        if self.api_version >= Version("20.12"):
+            return self.post("/logout")
+        else:
+            return self.get("/logout")
+
+    def __enter__(self):
+        return self
+
+    def __exit__(self, *exc):
+        super().__exit__()
```

Comment:
This closes transport adapters (not sure if it has any noticable effect) but sending logout request after this statement will possibly reopen them.
https://docs.python-requests.org/en/v1.2.3/api/#requests.adapters.HTTPAdapter